docs/docs-content/troubleshooting/edge/collect-support-bundles.md

```diff
@@ -0,0 +1,149 @@
+---
+
+When deploying or operating Edge clusters, you may encounter issues that require assistance from the Spectro Cloud
+support team, such as Edge hosts failing to register, pod failures, or provisioning errors. To help our team
+troubleshoot these issues more effectively, follow this guide to collect support bundles and attach them to your support
+ticket.
+
+## Limitations
+
+- This guide applies only to connected installation mode, not airgap environments, since it requires internet access to
+  run the troubleshooting script.
+
+## Prerequisites
+
+- You have connected to the host you need to troubleshoot, using SSH, direct terminal access, or another connection
+  method.
+- The following tools are available on the host:
+  - `journalctl`
+  - `systemctl`
+  - (Optional) `kubectl` (required for the full output of `support-bundle-infra.sh`)
+- (Optional) A valid `kubeconfig` file is configured (required for the full output of `support-bundle-infra.sh`)
+
+## Run the Troubleshooting Scripts
+
+### Run the Script to Collect Cluster Logs
+
+The `support-bundle-infra.sh` script collects cluster-level diagnostics. Follow the steps below on a host with a valid
+`kubeconfig` and `kubectl` installed, typically a control plane node.
+
+1. Set the path to your `kubeconfig` file.
+
+   ```bash
+   export KUBECONFIG=/etc/kubernetes/admin.conf
+   ```
+
+2. Run the `support-bundle-infra.sh` script.
+
+   ```bash
+   curl --silent --show-error --location https://software.spectrocloud.com/scripts/support-bundle-infra.sh | bash
+   ```
+
+   If the script runs successfully, it creates a file in your home directory. The file name follows the pattern
+   `<cluster name>-<YYYY-MM-DD>_<HH_MM_SS>.tar.gz`. If the script is unable to determine the cluster name, it defaults
+   to `spectro-cluster`. One of the messages the terminal displays contains the name of the created file.
+
+   ```bash
+   Logs are archived in spectro-cluster-2025-05-13_16_50_50.tar.gz
+   ```
+
+3. Copy the file to your local computer. The command below serves as an example of how to copy the file
+   `spectro-cluster-2025-05-13_16_50_50.tar.gz` from the `kairos@10.10.150.150` host to your `Downloads` folder. Run the
+   command on the machine to which you want to copy the file.
+
+   ```bash
+   scp kairos@10.10.150.150:~/spectro-cluster-2025-05-13_16_50_50.tar.gz ~/Downloads/
+   ```
+
+### Run the Script to Collect Edge Host Logs
+
+The `support-bundle-edge.sh` script collects host-level diagnostics from an Edge node and, if available, includes
+Kubernetes data accessible locally on that host. Follow the steps below on every Edge host you want to troubleshoot.
+
+1. (Optional) Set the path to your `kubeconfig` file.
+
+   ```bash
+   export KUBECONFIG=/etc/kubernetes/admin.conf
+   ```
+
+   If you do not perform this step, the script defaults to using `/run/kubeconfig` if it exists.
+
+2. Run the `support-bundle-edge.sh` script as a user with sudo privileges.
```

Comment:
```suggestion
2. Run the `support-bundle-edge.sh` script as a user with root privileges.
```
It's better to use English words whenever possible. You can use root or superuser (sudo is superuser do)